bin/install_deps.pl

```diff
@@ -451,10 +451,12 @@ sub name_overrides {
 # couple rules. When that doesn't work, add entries here for FreeBSD (port),
 # MacPorts ($dport), yum, and apt.
     my @modules = (
-        { module=>'LWP::UserAgent', info => { cat=>'www', port=>'p5-libwww', dport=>'p5-libwww-perl', yum=>'perl-libwww-perl' }, },
-        { module=>'Mail::Send'    , info => { port => 'Mail::Tools', }  },
-        { module=>'Date::Parse'   , info => { port => 'TimeDate',    }  },
-        { module=>'LWP'           , info => { port => 'p5-libwww',   }  },
+        { module=>'LWP::UserAgent'    , info => { cat=>'www', port=>'p5-libwww', dport=>'p5-libwww-perl', yum=>'perl-libwww-perl' }, },
+        { module=>'Mail::Send'        , info => { port => 'Mail::Tools', }  },
+        { module=>'Date::Parse'       , info => { port => 'TimeDate',    }  },
+        { module=>'LWP'               , info => { port => 'p5-libwww',   }  },
```

Comment:
@msimerson or you mean if LWP and LWP::UserAgent are installed by a single apt package? I think they're provided by `lib-www-perl`;

```
root@trusty:~# dpkg -S /usr/share/perl5/LWP
libwww-perl, liblwp-mediatypes-perl, liblwp-protocol-https-perl: /usr/share/perl5/LWP
```
```
root@trusty:~# dpkg -S /usr/share/perl5/LWP/UserAgent.pm 
libwww-perl: /usr/share/perl5/LWP/UserAgent.pm
```
```
root@trusty:~# dpkg -l | grep -i lwp
ii  liblwp-mediatypes-perl             6.02-1                           all          module to guess media type for a file or a URL
ii  liblwp-protocol-https-perl         6.04-2ubuntu0.1                  all          HTTPS driver for LWP::UserAgent
```
```
root@trusty:~# dpkg -L liblwp-protocol-https-perl 
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/liblwp-protocol-https-perl
/usr/share/doc/liblwp-protocol-https-perl/changelog.Debian.gz
/usr/share/doc/liblwp-protocol-https-perl/copyright
/usr/share/perl5
/usr/share/perl5/LWP
/usr/share/perl5/LWP/Protocol
/usr/share/perl5/LWP/Protocol/https.pm
/usr/share/man
/usr/share/man/man3
/usr/share/man/man3/LWP::Protocol::https.3pm.gz
```
```
root@trusty:~# dpkg -L liblwp-mediatypes-perl 
/.
/usr
/usr/share
/usr/share/perl5
/usr/share/perl5/LWP
/usr/share/perl5/LWP/MediaTypes.pm
/usr/share/perl5/LWP/media.types
/usr/share/man
/usr/share/man/man3
/usr/share/man/man3/LWP::MediaTypes.3pm.gz
/usr/share/doc
/usr/share/doc/liblwp-mediatypes-perl
/usr/share/doc/liblwp-mediatypes-perl/copyright
/usr/share/doc/liblwp-mediatypes-perl/changelog.Debian.gz
```